releases/release-announcement-template.md

```diff
@@ -0,0 +1,61 @@
+# Announcing `bindgen` $TODO_VERSION
+
+`bindgen` automatically generates Rust FFI bindings to C and C++ libraries.
+
+* [GitHub][]
+* [crates.io][]
```

Comment:
@emilio: when the slug matches the link's text, you can abbreviate to `[blah][]`. Pretty handy!

@nrc: great idea!